### PR TITLE
Level 2 rules for CIS

### PR DESCRIPTION
**Pull request by Achilleasein**
#### Description:

- _Implementing the following CIS server lvl 2 tests require_emergency_target_auth and require_singleuser_auth_

### CODE REVIEW

**Comment by Achilleasein** [comment]
We can probably squish the tests into one.

**Comment by jan-cerny** [comment]
Hello, please follow https://chris.beams.io/posts/git-commit/.

**Comment by matusmarhefka** [comment]
@Achilleasein Actually your test scripts are in wrong directory (`bash`), they should go into `tests` directory.

File: linux_os/guide/system/accounts/accounts-physical/require_emergency_target_auth/bash/cis_rhel7_require_emergency_target_auth.fail.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```
Comment (matusmarhefka) [inline linux_os/guide/system/accounts/accounts-physical/require_emergency_target_auth/bash/cis_rhel7_require_emergency_target_auth.fail.sh:1]: I think we don't need another fail scenario, there is already one and we can make it applicable to all RHEL versions by modifying `platforms` metadata to contain `multi_platform_rhel`. (pass scenarios still need to be separate for RHEL7 and RHEL8).

File: linux_os/guide/system/accounts/accounts-physical/require_emergency_target_auth/bash/cis_rhel7_require_emergency_target_auth.pass.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+#
+# platform = Red Hat Enterprise Linux 7
+# profiles = xccdf_org.ssgproject.content_profile_cis
```
Comment (matusmarhefka) [inline linux_os/guide/system/accounts/accounts-physical/require_emergency_target_auth/bash/cis_rhel7_require_emergency_target_auth.pass.sh:4]: I think we can remove this, it should work for any profile selecting this rule.

**Review by matusmarhefka — CHANGES_REQUESTED** [review]

File: linux_os/guide/system/accounts/accounts-physical/require_emergency_target_auth/bash/cis_rhel7_require_emergency_target_auth.fail.sh (hunk 1 above)
Comment (matusmarhefka) [reply linux_os/guide/system/accounts/accounts-physical/require_emergency_target_auth/bash/cis_rhel7_require_emergency_target_auth.fail.sh:1]: Pretty much same applies also to the `require_singleuser_auth` rule.

**Comment by matusmarhefka** [comment]
Please move your changes in your fork from master into another branch, I would also change the name of the test scripts, they don't need to contain rule ID, file names are missing pass/fail at the end and also remove `cis` from the file name as they are applicable also to other profiles. So change it to something like `rhel7_correct_value.pass.sh`.

**Pull request closed without merging**